packages/headless/src/features/search-and-folding/search-and-folding-request.test.ts

```diff
@@ -128,12 +168,20 @@ describe('buildSearchAndFoldingLoadCollectionRequest', () => {
     });
 
     it('#visitorId is not included in the request', async () => {
-      const request = await buildSearchAndFoldingLoadCollectionRequest(state);
+      const request = buildSearchAndFoldingLoadCollectionRequest(
+        state,
+        {location: '', referrer: '', userAgent: ''},
```

Comment:
It could be more flexible to convert this object into a mock (e.g., `buildMockNavigatorContext`) so that if it changes, one only needs to update a single place